src/app/components/ContractVerificationIcon/index.tsx

```diff
@@ -105,10 +107,12 @@ export const VerificationIcon: FC<{ address_eth: string; verified: boolean; noLi
                   rel="noopener noreferrer"
                   target="_blank"
                   sx={{ fontWeight: 400, color: 'inherit', textDecoration: 'underline' }}
+                  onClick={verified ? undefined : () => setExplainDelay(true)}
                 />
               ),
             }}
-          />
+          />{' '}
+          {explainDelay && t('contract.verification.explainVerificationDelay')}
```

Comment:
I would just show additional description always for not verified contract in details page. Not sure sure users will catch different copy when they switch context to another tab.

Reply:
yeah :/ but it doesn't apply 99.99% of the time

Reply:
Maybe if worded like: "Verify through Sourcify and give it a minute."